### ISSUE TITLE
runtime error: index out of range [3940] with length 355

### ISSUE THREAD

**Issue by YayEC** [hectorgimenez/koolo#300]
### Details

Running Andariel. Killing, pick some items, back to town, sell them and stopped action stands by Akara. No move, nada (counter runs reset, Debug/Settings/Start buttons only active. Game need to be closed and bot restarted. Happens after few runs, sometimes dozens. No pattern. Every time exact same error code. Not spotted when set and starts other runs except Andy. Log attached.

### Version

v0.4.0

### Logs

[ERROR.txt](https://github.com/user-attachments/files/15834835/ERROR.txt)

**Comment by tlbbpcg** [comment]
yes, I've also encountered this issue, whatever hope bot can restart when meet same error.
